interface/orders/types_edit.php

```diff
@@ -85,10 +85,9 @@ function recursiveDelete($typeid)
 <html>
```

Comment:
The ? icon tooltip here does nothing

Reply:
Update the commit.
![image](https://user-images.githubusercontent.com/22230889/92089784-359bc380-ee01-11ea-9e54-84089bdd083c.png)